### PR TITLE
wg-security-audit: add

### PR DESCRIPTION
**Pull request by jessfraz**
related to: https://github.com/kubernetes/steering/issues/71

you can find our charter here: https://docs.google.com/document/d/1RbC4SBZBlKth7IjYv_NaEpnmLGwMJ0ElpUOmsG-bdRA/edit

we will reuse the sig-auth zoom for meetings so others can join cc @aasmall @joelsmith

### CODE REVIEW

File: wg-security-audit/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Organizers
+
+* Jessie Frazelle (**[@jessfraz](https://github.com/jessfraz)**), Microsoft
+* Aaron Small (**[@aasmall](https://github.com/aasmall)**), Google
+* Joel Smith (**[@joelsmith](https://github.com/joelsmith)**), Red Hat
+
+## Contact
+* [Slack](https://kubernetes.slack.com/messages/)
+* [Mailing list]()
```
Comment (alban) [inline wg-security-audit/README.md]: @jessfraz the slack/ML links are missing.

Comment (jessfraz) [reply wg-security-audit/README.md]: yeah we are going to use kubernetes-security-discuss@googlegroups.com and no slack for now unless people want it, just more to cleanup when we are done, do you think we need one?

Comment (alban) [reply wg-security-audit/README.md]: I think it's fine not to have a dedicated channel for the wg: there is a slack channel "#kubernetes-security" too. I was confused that the links were not working and I found out later that this page is generated from a template.
Thanks for the link - I just joined the mailing list.

**Pull request merged**